src-plugins/itkFilters/itkFiltersProcessBase.cpp

```diff
@@ -89,3 +88,23 @@ void itkFiltersProcessBase::eventCallback ( itk::Object *caller, const itk::Even
 
     source->emitProgress((int) (processObject->GetProgress() * 100));
 }
+
+dtkSmartPointer<medAbstractImageData> itkFiltersProcessBase::getInputData()
+{
+    return d->inputData;
+}
+
+void itkFiltersProcessBase::setInputData(dtkSmartPointer<medAbstractImageData> inputData)
```

Comment:
there is only one process which erase the input, but the original setting of the input is done in `itkFiltersProcessBase::setInput(medAbstractData *data, int channel)`